src/components/SpinBox.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { KeyboardEvent, ReactElement, useCallback, useEffect, useState } from "react";
+import styles from "./SpinBox.module.css";
+
+/** Path for the SVG of the spin box handles */
+const spinBoxHandlePath =
```

Comment:
how was this generated?  Should it live in its own svg file as an asset?

Reply:
This was copied off of the Figma doc! I agree that this could be moved to a separate file; I can do that for the other SVG files in the app too.